connect/go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/abrander/garmin-connect/connect
+
+go 1.16
```

Comment:
I don't think Go 1.16 is needed? At least 1.15 should suffice, probably older too, but I have no (easy) way of testing.

Reply:
I will change it to 1.15. It is autogenerated by go mod init.